starknet/tests/eth-sig-auth.test.ts

```diff
@@ -0,0 +1,258 @@
+import fs from 'fs';
+import { ethers } from 'ethers';
+import { Provider, Account, CallData, cairo, json } from 'starknet';
+import {
+  Propose,
+  proposeTypes,
+  voteTypes,
+  Vote,
+  updateProposalTypes,
+  UpdateProposal,
+} from './types';
+
+const pk = process.env.PRIVATE_KEY || '';
+
+describe('Starknet Signature Authenticator', () => {
+  const signer = new ethers.Wallet(pk);
+  const provider = new Provider({ sequencer: { baseUrl: 'http://127.0.0.1:5050' } });
```

Comment:
should this be a constant defined somewhere?

Reply:
ive drawn network url from env var